### PR TITLE
Fix settings.xml in M1 Mac Support Section

### PR DESCRIPTION
**Pull request by jugomezv**

### CODE REVIEW

**Review by snleee — APPROVED** [review]

File: basics/getting-started/running-pinot-locally.md [hunk 1]
```diff
@@ -82,15 +82,15 @@ Currently Apache Pinot doesn't provide official binaries for M1 Mac. You can how
   <activeProfile>
     apple-silicon
   </activeProfile>
+  <profiles>
```
Comment (Jackie-Jiang) [inline basics/getting-started/running-pinot-locally.md:85]: Is this correct? Based on my understanding, `profiles` needs to be a separate section

Comment (snleee) [reply basics/getting-started/running-pinot-locally.md:85]: According to @jugomezv, he was able to get it working with the above configs. @jugomezv Can you confirm on this? 

I personally don't have m1 macbook so I cannot confirm this.

Comment (snleee) [reply basics/getting-started/running-pinot-locally.md:85]: Maybe we need to revert this change and wrap everything under `<settings></settings>`?

I found some examples below
https://github.com/grpc/grpc-java/issues/7690

Comment (jugomezv) [reply basics/getting-started/running-pinot-locally.md:85]: OK we tried the corrected version as proposed by @snleee and that works properly, I will edit the PR accordingly. Here is the error message I get with the current settings.xml:

```
[ERROR] Error executing Maven.
[ERROR] 1 problem was encountered while building the effective settings
[FATAL] Non-parseable settings /Users/jugomez/.m2/settings.xml: start tag not allowed in epilog but got p (position: END_TAG seen ...</activeProfiles>\n  <p... @6:5)  @ /Users/jugomez/.m2/settings.xml, line 6, column 5
```

**Review by snleee — APPROVED** [review]
LGTM

@Jackie-Jiang Can you review this again?

**Pull request merged**